.github/workflows/ci.yaml

```diff
@@ -0,0 +1,43 @@
+name: ci
+
+on:
+  push:
+    branches: [main, actions-test]
+  pull_request:
+    branches: [main, actions-test]
+
+jobs:
+  build_and_test:
+    name: build and test (Elixir ${{matrix.elixir}} | Erlang/OTP ${{matrix.otp}})
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - elixir: 1.12.x
+            otp: 24.x
+          - elixir: 1.13.x
+            otp: 24.x
+          - elixir: 1.14.x
+            otp: 24.x
+          - elixir: 1.13.x
+            otp: 25.x
+          - elixir: 1.14.x
+            otp: 25.x
```

Comment:
can you copy the same  matrix as [here](https://github.com/mhanberg/gen_lsp/blob/main/.github/workflows/ci.yml#L12-L24) ?

that is the test suite that credo uses, so it would be good to test it here.